libs/akita-ng-effects/src/lib/akita-ng-effects.module.ts

```diff
@@ -35,6 +35,7 @@ export class AkitaNgEffectsModule {
         {
           provide: _FEATURE_EFFECTS,
           useValue: featureEffects,
+          multi: true,
```

Comment:
Did you check this? `multi` will change it to be an array, so it should be a multi-dimensional array now.

Reply:
Let me check again.

Reply:
It is. You're correct. I changed it.